### ISSUE TITLE
 Change any references to old Restaurant Recommender project (blocked by #797)

### ISSUE THREAD

**Issue by alodahl** [Techtonica/curriculum#799]
Note: Blocked until #797 is finished.
## Task: 
- [ ] In #797 , we changed the Restaurant Recommender project into an Event search project. Your task is to search the Techtonica GitHub project for any reference to the Restaurant Recommender project or its 2 files and make sure that the Event search project is mentioned instead.

- [ ] Make sure the [Readme links](https://github.com/Techtonica/curriculum#client-side-development--confident-programming) are updated.

- [ ] Make sure [the Roadmap](https://docs.google.com/spreadsheets/d/1vIr3fiKa_jxzy7qtFsiE5hEJbiTXwc1LYxbfsHOYD6U/edit?usp=sharing) is updated. Ask Alina for edit permissions.

- [ ] Do a search in the rest of the Techtonica repo files for "restaurant", restaurant recommender", and their two file names to make sure they're all updated.

**Comment by yati1998** [comment]
@alodahl  I would really like to work on this issue once you accept my changes on the previous PRs. It will be great of you if you assign me this issue when you feel, I am eligible for it.
Thank You.

**Comment by alodahl** [comment]
I just assigned you another issue, @yati1998 .  Do you want to switch?

**Comment by yati1998** [comment]
No I have worked on that PR.  I just want to work on this issue once the other is approved.  It's ohk if you assign me this issue later.

**Comment by alodahl** [comment]
Ok, just be aware that by finished, here it means merged.

**Comment by raibove** [comment]
@alodahl can I work on this issue.

**Comment by raibove** [comment]
@alodahl please give me access to edit Roadmap

**Comment by raibove** [comment]
I don't have write access to README file. It created conflicts to my PR. Please provide me access to README.md

**Comment by alodahl** [comment]
Your write access and your conflicts are not related. Resolve the conflicts per my PR comment and it should be fine.

**Comment by alodahl** [comment]
Slack me your email address so I can give you access to the Google Sheet Roadmap.

**Comment by raibove** [comment]
I have doubt regarding updating roadmap. Do I need to change the link from line 127,131 and 136.
Will there be any change in the order of week 5 & 6.

**Comment by alodahl** [comment]
You should only be replacing lines with restaurant recommender links. That is the only change.

**Comment by raibove** [comment]
Ok. I made the changes in sheet

**Comment by alodahl** [comment]
Thank you! 😄
We’re you  able to complete all the tasks in the description?

**Comment by raibove** [comment]
Yes
I updated the readme and roadmap. I also looked for restaurant recommender project in the curriculum but it was not mentioned
